Makefile

```diff
@@ -153,20 +154,30 @@ batcher_send_plonk_bls12_381_task: batcher/client/target/release/batcher-client
 
 batcher_send_groth16_bn254_task: batcher/client/target/release/batcher-client
 	@echo "Sending Groth16Bn254 1!=0 task to Batcher..."
-	@cd batcher/client/ && cargo run --release -- \
+	@cd batcher/client/ && cargo run --release --  \
 		--proving_system Groth16Bn254 \
-		--proof test_files/groth16/ineq_1_groth16.proof \
-		--public_input test_files/groth16/ineq_1_groth16.pub \
-		--vk test_files/groth16/ineq_1_groth16.vk
+		--proof ../../test_files/groth16/ineq_1_groth16.proof \
+		--public_input ../../test_files/groth16/ineq_1_groth16.pub \
+		--vk ../../test_files/groth16/ineq_1_groth16.vk \
+
+batcher_send_plonk_task: batcher/client/target/release/batcher-client
```

Comment:
this target should require the files to be generated, or should be stored in the repo

Reply:
Those should be stored in the same manner as the ones from Groth16.